public/sw.js

```diff
@@ -12,22 +12,27 @@ self.addEventListener('install', event => {
 
 const fetchEventHandler = async event => {
   if (
-    /^https:\/\/ichef(\.test)?\.bbci\.co\.uk\/(news|ace\/(standard|ws))\/.+(\.jpg|\.png)$/.test(
+    /^https:\/\/ichef(\.test)?\.bbci\.co\.uk\/(news|images|ace\/(standard|ws))\/.+.webp$/.test(
       event.request.url,
     )
   ) {
     const req = event.request.clone();
 
     // Inspect the accept header for WebP support
-    let supportsWebp = false;
-    if (req.headers.has('accept')) {
-      supportsWebp = req.headers.get('accept').includes('webp');
-    }
 
-    // If we support WebP
-    if (supportsWebp && !/\/amz\/worldservice\/.*/.test(event.request.url)) {
+    const supportsWebp =
+      req.headers.has('accept') && req.headers.get('accept').includes('webp');
+
+    // if supports webp is false in request header then don't use it?
```

Comment:
```suggestion
    // if supports webp is false in request header then don't use it
```

Guess this isn't a question?